### DISCUSSION TITLE
Error running Bot.py ('NoneType' object is not subscriptable)

### DISCUSSION THREAD

**Discussion by SALBO-Y** [conor19w/Binance-Futures-Trading-Bot#19] [Q&A]
Thank you conor19w

After running a bot.py.
Print a my account balance and then error is print 
plz help me the solution... 

<Error type> 
Combining Historical and web socket data...
Finished.
Start Balance: xx.xxxx
**'NoneType' object is not subscriptable
<class 'TypeError'> Bot.py 65**

**Comment by conor19w** [comment]
In make_Decision() in Data_set.py do you return anything?
